syncer/online_ddl.go

```diff
@@ -250,6 +252,11 @@ func (s *OnlineDDLStorage) Clear() error {
 	return nil
 }
 
+// ResetConn implements CheckPoint.ResetConn
```

Comment:
incorrect comment. Did you copy it from _checkpoint.go_?

Reply:
[fix it](https://github.com/pingcap/dm/pull/266/files#diff-18d840e544a27bd6ebe280a590e1dbf9R255)